client/state/selectors/index.js

```diff
@@ -45,3 +45,5 @@ export isSharingButtonsSaveSuccessful from './is-sharing-buttons-save-successful
 export isSiteBlocked from './is-site-blocked';
 export isSiteSupportingImageEditor from './is-site-supporting-image-editor';
 export isTransientMedia from './is-transient-media';
+export isRequestingReaderTeams from './is-requesting-reader-teams';
+export getReaderTeams from './get-reader-teams';
```

Comment:
+1.

I've just fixed the order of the teams constants in `action-types.js` while rebasing #10777.

I'll whip up a quick PR to fix the order of the selectors include 👍